DiepSam_html/index.html

```diff
@@ -0,0 +1,286 @@
+          </div>
+          <div class="block-news col-lg-4 col-md-4">
+            <img src="images/event2.jpg">
+            <strong><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt</p></strong>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident</p>
+          </div>
+          <div class="block-news col-lg-4 col-md-4">
+            <img src="images/event3.jpg">
+            <strong><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt</p></strong>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="gallery">
+      <div class="container">
+        <div class="section-head">
+          <h3 class="section-title">gallery</h3>
+          <div class="logo">
+            <img src="images/icon.png" alt="logo icon">
+          </div>
+        </div>
+        <div class="info-gallery col-lg-12 col-lg-12">
+          <div class="col-lg-3 col-md-3">
+            <div class="block-gallery">
+              <img src="images/gal1.jpg">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+          </div>
+          <div class="col-lg-3 col-md-3">
+            <div class="block-gallery">
+              <img src="images/gal2.jpg">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+          </div>
+          <div class="col-lg-3 col-md-3">
+            <div class="block-gallery">
+              <img src="images/gal3.jpg">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+          </div>
+          <div class="col-lg-3 col-md-3">
+            <div class="block-gallery">
+              <img src="images/gal4.jpg">
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="map-daNang">
```

Comment:
- Create new section `contact` and put `map` , `contact us`, `form`, `company` to here.
- Only need name class `map` .. no need `map-daNang`
- Never add name class like this. `daNang`